rfcs/convergence/global-styles.md

```diff
@@ -0,0 +1,50 @@
+# RFC: Global styles
+
+---
+
+@miroslavstastny
+
+## Summary
+
+Fluent UI Converged components' styles expect global styles to be set. This RFC proposes how those global styles will be injected into DOM.
+
+## Background
+
+There are two kinds of global (not owned by a component) styles which the converged components depend on - **CSS resets** and **Theme root styles**.
+
+### Theme root styles
+
+## Detailed Design or Proposal
+
+### CSS resets
+
+Converged components depend on Normalize.css for CSS resets. Fluent UI needs to provide a way how an application can inject the Normalize styles in each `window` instance.
+
+Both v7/v8 and v0/northstar components require Normalize.css as well. During a transition phase, converged components will be used side by side with existing components. In these cases Normalize.css will be handled out of converged components, therefore converged must have a possibility to opt-out of injecting the Normalize.
```

Comment:
```suggestion
Both v7/v8 and v0/Northstar components require `normalize.css` as well.

- v8 **A LINK FOR V8**
- v0 [normalizeCSS.ts](https://github.com/microsoft/fluentui/blob/31ff08b1d7ca1aded8f2f42d5d90fbb3146b5549/packages/fluentui/react-northstar/src/themes/teams/staticStyles/normalizeCSS.ts)
 
During a transition phase, converged components will be used side by side with existing components. In these cases `normalize.css` will be handled out of converged components, therefore converged must have a possibility to opt-out of injecting the Normalize.
```

It will nice to reference files, too